fvm/context.go

```diff
@@ -59,9 +60,18 @@ func newContext(ctx Context, opts ...Option) Context {
 const AccountKeyWeightThreshold = 1000
 
 const (
-	DefaultGasLimit                     = 100_000 // 100K
-	DefaultEventCollectionByteSizeLimit = 256_000 // 256KB
+	DefaultComputationLimit             = 100_000     // 100K
+	DefaultMemoryLimit                  = 100_000_000 // 100G
```

Comment:
Is this per transaction? This seems very high, no? Or is it intentionally high until we find "average" / "normal" uses?

Reply:
yeah, I set it to a large number until we realize what should be actual values, also don't think its connected so, meterMemory calls are not received yet.

Reply:
I was worried to replace them with math.MaxUint64

Reply:
set it to 2GB for now, but it's not enforced since there is no call to meter memory yet. would adjust in that PR